cmd/ftl/cmd_new.go

```diff
@@ -30,9 +31,10 @@ type newCmd struct {
 }
 
 type newGoCmd struct {
-	Replace map[string]string `short:"r" help:"Replace a module import path with a local path in the initialised FTL module." placeholder:"OLD=NEW,..." env:"FTL_INIT_GO_REPLACE"`
-	Dir     string            `arg:"" help:"Directory to initialize the module in."`
-	Name    string            `arg:"" help:"Name of the FTL module to create underneath the base directory."`
+	Replace   map[string]string `short:"r" help:"Replace a module import path with a local path in the initialised FTL module." placeholder:"OLD=NEW,..." env:"FTL_INIT_GO_REPLACE"`
+	Dir       string            `arg:"" help:"Directory to initialize the module in."`
+	Name      string            `arg:"" help:"Name of the FTL module to create underneath the base directory."`
+	GoVersion string
```

Comment:
Delete

Reply:
I added this one so that it could be used in the scaffolding template since that uses `newGoCmd` directly as it's context. Would it be better to create a new context and add what I need there for the template instead of using the `newGoCmd` for template context?

Reply:
Yes, if you need it in the context but it's not a flag, it shouldn't be in the flag struct.